# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../udev/no_libvolume_id_so.diff
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./extras/volume_id/lib/Makefile.orig	2007-06-14 13:00:06.000000000 -0400
+++ ./extras/volume_id/lib/Makefile	2007-06-14 13:01:51.000000000 -0400
@@ -59,7 +59,7 @@
 	libvolume_id.h		\
 	util.h
 
-all: libvolume_id.a $(SHLIB) libvolume_id.pc
+all: libvolume_id.a libvolume_id.pc
 .PHONY: all
 .DEFAULT: all
 
@@ -103,10 +103,8 @@
 
 install: all
 	$(INSTALL_DATA) -D libvolume_id.h $(DESTDIR)$(includedir)/libvolume_id.h
-	$(INSTALL_LIB) -D $(SHLIB) $(DESTDIR)$(libdir)/$(SHLIB)
 	mkdir -p $(DESTDIR)$(usrlibdir)/
-	ln -sf $(SHLIB) $(DESTDIR)$(libdir)/libvolume_id.so.$(SHLIB_CUR)
-	ln -sf $(libdir)/$(SHLIB) $(DESTDIR)$(usrlibdir)/libvolume_id.so
+	$(INSTALL_LIB) -D libvolume_id.a $(DESTDIR)$(usrlibdir)/libvolume_id.a
 	$(INSTALL_DATA) -D libvolume_id.pc $(DESTDIR)$(usrlibdir)/pkgconfig/libvolume_id.pc
 .PHONY: install
 
